spec/support/unit/sensitive.rb

```diff
@@ -0,0 +1,58 @@
+# rubocop:disable all
```

Comment:
This file is a direct copy of what should be hidden in a gem in the bundle. One of the things that makes rubocop angry about this file is the way a couple of the methods are defined. There's no need to jump through hoops  to make rubocop happy when this file should be removed as soon as the bug in rspec-puppet is fixed.